.github/workflows/sphinx.yml

```diff
@@ -0,0 +1,29 @@
+name: Sphinx Pages
```

Comment:
Cool. And using this the sphinx sources somewhere within the master branch are generated and published to Github pages?

Reply:
yes.